52.md

```diff
@@ -0,0 +1,116 @@
+
+```json
+{
+  "id": <32-bytes lowercase hex-encoded sha256 of the the serialized event data>
+  "pubkey": <32-bytes lowercase hex-encoded public key of the app owner issuing the badge>,
+  "created_at": <unix timestamp in seconds>,
+  "kind": XX,
+  "tags": [
+    ["p", <32-bytes hex pubkey of the member receiving the badge>],
+    ["badge_type", <integer specifying badge type>]
+  ],
+  "content": <stringified json object, matching badge_type schema>,
+  "sig": <64-bytes signature of the sha256 hash of the serialized event data, which is the same as the "id" field>
+}
+```
+
+`p tag` MUST contain exactly one entry, the pubkey of the claim's subject.
+
+`badge_type` identifies badge type. Current accept values are `0 - Achievement Badge` and `1 - Membership Badge`
+
+Like Kind 0, badges provide metadata about a pubkey, but the event is sent (and thus signed) by the app owner, not the subject's pubkey.
+
+Two badge types are defined, with the possibility of adding more if required.
+
+### 0 - Achievement Badge
+Awarded to a user for an achievement. 
+
+Achievement Badge Schema
+```json
+{
+    "issuer_name": <string of app name or community name>,
+    "name": <string of achievement name>,
+    "message": <string of message>,
+    "image": <url of badge image>,
+    "score": <optional integer of score value>
+}
+````
+
+Example as content
+```json
+{
+    "content": "{\"issuer_name\": \"nostr.directory\", \"name\": \"Donation\", \"message\": \"User has sent a sats donation to noster.directory.\", \"image\": \"https://myapp.com/badges/donation.svg\", \"score\": 1000}",
+    ...
+}
+````
+
+- issuer_name - name of app or community. e.g. "Nosterplebs", "Continuous Delivery Foundation"
+- name - name of the achievement. e.g. "Donation", "Top Bug Crusher"
+- message - Text description of the achievement
+- image - url link to square badge image (1:1 ratio). Ideally .svg image, but if not SHOULD be 32 x 32 pixels.
+- score - optional integer value. Include when you need to rank achievements. e.g. Display "High Score" list in descending order
+
+Client may include additional key/value pairs in content.
+
+### 1 - Membership Badge
+Sent by app owner to confirm that a user is the member of app, community or group.
+
+Membership Badge Schema
+```json
+{
+    "issuer_name": <string of app name or community name>,
+    "image": <url of badge image>,
+    "role": <optional string of role of user within the community>,
+    "id": <optional string of unique user identity>,
+    "alias": <optional string ofuser alias>,
+    "member_since": <optional unix timestamp of when member joined>,
+    "expiry_date": <optional unix timestamp of membership expiry date>,
+    "profile": <optional url to user profile page in app>,
+    "followers": <optional integer count of followers if applicable>,
+    "connections": <optional integer count of connections if applicable>,
+    "posts": <optional integer count of posts if applicable>
+}
```

Comment:
Many of these fields are app-specific. Reduce to just the fields that should be specified in the protocol. Keep allowance that clients than add their own fields.